pyvista/plotting/lookup_table.py

```diff
@@ -748,7 +750,7 @@ def apply_opacity(self, opacity, interpolate: bool = True, kind: str = 'quadrati
         """
         if isinstance(opacity, (float, int)):
```

Comment:
It might be a bit surprising that ints are interpreted as floats, but I guess the reason for this is that this is short-hand for passing 1 instead of 1.0. Since we raise in case of ints greated than 1 this is probably fine, because people passing 1 probably don't mean 1/255.